### PR TITLE
[Project Submission] Create NiftySubs.md

### PR DESCRIPTION
**Pull request by therealharpaljadeja**

### CODE REVIEW

File: projects/NiftySubs.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  
+  #### Huy Vu Nguyen
+  Product and Design in the web3 ecosystem
+  
+  #### Rishabh Keshan
+  Mern Stack Developer, React Native, Web3, UI/UX Developer
+
+
+## How the community can engage
+* GitHub Discussion: <!--Start a discussion with the community here: https://github.com/ipfs/community/discussions/new and attach the link!-->
```
Comment (autonome) [inline projects/NiftySubs.md:88]: Please create a discussion thread and link it here

**Review by autonome — CHANGES_REQUESTED** [review]
One minor change and can merge!

File: projects/NiftySubs.md (hunk 1 above)
Comment (therealharpaljadeja) [reply projects/NiftySubs.md]: Github Discussion Link Added!

**Review by therealharpaljadeja — COMMENTED** [review]
Github Discussion link added!

**Pull request merged**